recipes/MADL-CPU-IntelMPI/README.md

```diff
@@ -0,0 +1,59 @@
+# MADL-CPU-OpenMPI
+This recipe shows how to run Microsoft Azure Distributed Linear Learner (MADL) on CPUs across
+Azure VMs via Open MPI.
+
+## Configuration
+Please see refer to this [set of sample configuration files](./config) for
+this recipe.
+
+### Pool Configuration
+The pool configuration should enable the following properties:
+* `vm_size` should be a CPU-only instance, 'STANDARD_D2_V2'.
+* `inter_node_communication_enabled` must be set to `true`
+* `max_tasks_per_node` must be set to 1 or omitted
+* `publisher` should be `Canonical` 
+* `offer` should be `UbuntuServer`
+* `sku` should be `7.3`
+
+### Global Configuration
+The global configuration should set the following properties:
+* `docker_images` array must have a reference to a valid MADL
+Docker image that can be run with OpenMPI. The image denoted with `0.0.1` tag found in [msmadl/symsgd:0.0.1](https://hub.docker.com/r/msmadl/symsgd/)
+is compatible with Azure Batch Shipyard VMs. 
+
+### MPI Jobs Configuration (MultiNode)
+The jobs configuration should set the following properties within the `tasks`
+array which should have a task definition containing:
+* `docker_image` should be the name of the Docker image for this container invocation.
+For this example, this should be
```

Comment:
`this can be ..., but use the latest tagged version available.` (if you're omitting the versioning above).

Reply:
ok